app/models/authorization.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
+
+class Authorization
+  attr_accessor :token
+  attr_reader :token_auth_accounts
+
+  def initialize(token)
+    @token = token
+  end
+
+  def id
```

Comment:
if it is the `self` link, which is all I can think of, we can override the `self_link` method on the representer to not need this.